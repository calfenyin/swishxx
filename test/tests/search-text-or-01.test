search | | -i text.index | abominable or license | 0
