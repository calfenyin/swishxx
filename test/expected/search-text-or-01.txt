# results: 3
100 ./Time_Machine,_The.txt 182203 Time_Machine,_The.txt
42 ./GNU_GPLv2.txt 17982 GNU_GPLv2.txt
36 ./Gutenberg_License.txt 17308 Gutenberg_License.txt
